packages/flame/lib/src/game/flame_game.dart

```diff
@@ -44,6 +45,9 @@ class FlameGame extends Component with Game {
   @override
   Vector2 get size => camera.gameSize;
 
+  @override
+  ComponentTreeRoot get fcsRoot => this;
```

Comment:
Maybe it could just be called `root`, since we don't prefix anything else in the system with `fcs`?

Reply:
My thinking here was like this:
- this is a deeply internal property that will probably not be of interest to the users.
- at the same time, it is common for users to add lots of their own properties into their derived Game classes. As such, if this was simply `root`, there would be a risk of a name clash for some users.

Reply:
I think it is better for us to take the cleaner word `root` anyways, even though it might clash with any user definitions, they users can easily add a more descriptive name to their variable too.

It is not that bad here, but I just don't want it to spread so that we start prepending fcs to all kinds of things, so prefer killing the convention in its cradle. 😅